recipes/norm/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+import yaml
+
+
+class NormConan(ConanFile):
+    name = "norm"
+    description = "A reliable multicast transport protocol"
+    topics = ("conan", "norm", "multicast", "transport protocol")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.nrl.navy.mil/itd/ncs/products/norm"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    license = "NRL"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["NORM_CUSTOM_PROTOLIB_VERSION"] = "./protolib" # FIXME: use external protolib when available in CCI
```

Comment:
protokit seems to silently try find libxml2. We might prefer to disable this behavior (or add an option to enable/disbale libxml2 dependency) as it leads to unpredictable build depending on build platform:
https://github.com/USNavalResearchLaboratory/protolib/blob/702e909945446275f0313846e988612883fca94a/CMakeLists.txt#L89

Reply:
Maybe you should
- add `libxml2/2.9.10` dependency unconditionally for the moment (with a small comment explaining that it's a dependency of protokit).
- add `cmake_find_package_multi` generator in conanfile.py
- replace `list(APPEND PLATFORM_LIBS ${LIBXML2_LIBRARIES})` in protokit CMakeLists by `list(APPEND PLATFORM_LIBS LibXml2::LibXml2)`